src/Utilities/results.jl

```diff
@@ -37,17 +37,21 @@ the `ObjectiveFunction` value.
 """
 function get_fallback(model::MOI.ModelLike, attr::MOI.ObjectiveValue)
     MOI.check_result_index_bounds(model, attr)
-    F = MOI.get(model, MOI.ObjectiveFunctionType())
-    f = MOI.get(model, MOI.ObjectiveFunction{F}())
-    obj = eval_variables(model, f) do vi
-        return MOI.get(model, MOI.VariablePrimal(attr.result_index), vi)
-    end
-    if is_ray(MOI.get(model, MOI.PrimalStatus()))
-        # Dual infeasibility certificates do not include the primal
-        # objective constant.
-        obj -= MOI.constant(f, typeof(obj))
+    try
```

Comment:
Why is this needed ?

Reply:
I added the try defensively to all methods. But the issue is that `get_fallback` might be used by something that doesn't support getting the `ConstraintFunction. That might be because it isn't implemented, or it might be because the model uses a `ZerosBridge` (or similar). We should throw the proper error, instead of throwing whatever error actually threw, like "can't unbridge the Zeros bridge."

In the Cbc case, it's something like `CachingOptimizer(LazyBridgeOptimizer(CachingOptimizer(Cbc))`

The outer cache says "can you get ConstraintPrimal." The bridge says yes. Ends up erroring, falling back to `get_fallback`, and then errors when it tries to unbridge. 

If it thew `GetAttributeNotAllowed`, then the outer cache would fallback to `get_fallback` and this would succeed.

Even if it didn't have an outer cache to fall back do, we should throw the correct `GetAttributeNotAllowed` for consistency.

Reply:
Isn't this also fixed by throwing a `GetAttributeNotAllowed` for the ZerosBridge ?

Reply:
If we don't see the error of the zerosbridge then the user won't know he just has to remove this bridge, that's an issue

Reply:
What we could do is throw a custom error here:
https://github.com/jump-dev/MathOptInterface.jl/blob/0249cfa8400dd550527b41db942e85cf0626d417/src/Bridges/Variable/map.jl#L582-L586
Here we try catch:
https://github.com/jump-dev/MathOptInterface.jl/blob/0249cfa8400dd550527b41db942e85cf0626d417/src/Bridges/bridge_optimizer.jl#L2384
If we see exactly this error, we thro a `GetAttributeNotAllowed` for `ConstraintFunction` with the error message of:
https://github.com/jump-dev/MathOptInterface.jl/blob/0249cfa8400dd550527b41db942e85cf0626d417/src/Bridges/Variable/map.jl#L582-L586
otherwise, we rethrow

Reply:
This would address this particular issue with ZerosBridge, but it ignores with wider issue. `get_fallback` may fail, and instead of throwing a nice error, we may throw some arbitrary error. But perhaps you're right, we can just disable ZerosBridge for now in Cbc.

Reply:
The issue with this error is that it's difficult to debug since it provides no information on the initial error. Also, I'm not sure all the issues with `ZerosBridge` will go through `get_fallback` so I feel it's easier to fix at the source as I suggested in my previous comment